package.json

```diff
@@ -30,6 +31,7 @@
     "node": ">=4"
   },
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
Anything outside of `dependencies` would be preferable, since all instances don't really need the typescript definitions. Is there a way to have a ts-jquery-once package, that introduces the definitions, along with bringing in jQuery Once?